ParisBNF/abb/BNFabb147.xml

```diff
@@ -0,0 +1,522 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BNFabb147" xml:lang="en" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>A sawāsǝw-treatise on lexicon and grammar</title>
+            <editor role="cataloguer" key="DN"/>
+            <editor role="generalEditor" key="AB"/>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </p>
+               </licence>
+            </availability>
+            <date>2016-06-07T17:45:12.922+02:00</date>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                  <idno>BnF Éthiopien d'Abbadie 147</idno>
+                    </msIdentifier>
+
+                  <physDesc>
+                    <objectDesc form="Codex">
+                       <supportDesc>
+                          <support>
+                             <material key="parchment"/>
+                          </support>
+                          <extent>
+                            <measure unit="page">A+128</measure>
+                            <measure unit="leaf">A+64</measure>
```

Comment:
have you checked the guidelines and the issue https://github.com/BetaMasaheft/Documentation/issues/913

Some Manuscripts are paginated instead of being foliated. You will have in such cases most probably to be consistent and provide inside <gi>extent</gi> a <tag>measure unit="page"</tag>. This will then alter the behaviour of the visualization and use p. a pp. instead of f. and ff. . However, the values you MIGHT enter inside <att>from</att>, <att>to</att> or <att>target</att> are still the foliation references, so, 1v, not 2, 2v, not 4.The locus markup of a paginated manuscript will look like in the following example</p>
`<locus from="1ra" to="6vb">1-12</locus>`

Reply:
Ok, I tried.

Reply:
and delete line 42, leave only one `<measure unit="page">A+128</measure>`

Reply:
I understood that this is only for  collation, right?

Reply:
Done.

Reply:
the extent should give unit "page", then if you write e.g.
`<locus from="1ra" to="61vb">1a-122b</locus>`
this will print pp. 1a-122b, since these are page numbers, but compute correctly the references
this is the way the app is programmed at the moment

you do it correctly for unit 1 but the overall description as
```
   <measure unit="page">A+128</measure>
                            <measure unit="leaf">A+64</measure>
```
does not look correct, as I understand there are 128 pages in unit 1 and then additional unpaginated leaves in units 2-4, so probably correct would be, for the overall description,
```
   <measure unit="page">A+128</measure>
                            <measure unit="leaf">3</measure>
```
and then for unit 1 only    `<measure unit="page">A+128</measure>
`but i am not sure how the app deals with this

Reply:
Maybe, A+128+129-134?

Reply:
no you should give the total in the extent

Reply:
measure unit="page"> 136 (A+128+129-134)</measure>?

Reply:
what does this mean? how many pages and leaves are there?

Reply:
Ok, I see I misplaced my remark on the additional pages. I will correct it.

Reply:
I read the description as (2(A)+128)+2+2+2 = 136 in 4 units. what does 129-134 mean?

Reply:
(2(A)+128)+2+2+2 is correct. i missed 2 pages in the beginning